src/ape/managers/project/manager.py

```diff
@@ -157,6 +153,35 @@ def compiler_data(self) -> List[Compiler]:
 
         return compilers
 
+    @property
+    def meta(self) -> PackageMeta:
+        """
+        Metadata about the active project as per EIP
+        https://eips.ethereum.org/EIPS/eip-2678#the-package-meta-object
+        Use when publishing your package manifest.
+        """
+
+        return self.config_manager.meta  # type: ignore
+
+    @property
+    def package_deployments(self) -> Dict[BIP122_URI, Dict[str, EthPMContractInstance]]:
```

Comment:
why `package_deployments`? why not `deployments`?

Reply:
I am nervous a user will assume this is similar to `project.MyContract.deployments` where it is everything deployed.
I want to be sure that it is obvious as possible that these are only the publishing / final-package related deployments.

That being said, I am now realizing that `package_deployments` sounds like a deployment of a full package, so maybe we need a better name.

Reply:
Ideas so far:

```
disclosed_deployments
ethpm_deployments
publishable_deployments
package_deployments
build_deployments
manifest_deployments
```

Reply:
Renamed to `tracked_deployments` and added docs.